charts/consul/templates/ingress-gateways-deployment.yaml

```diff
@@ -399,6 +404,9 @@ spec:
                         -partition={{ $root.Values.global.adminPartitions.name }} \
                         {{- end }}
                         -id="${POD_NAME}"
+                  {{- if $root.Values.global.acls.manageSystemACLs }}
+                  - "/consul-bin/consul logout"
+                  {{- end}}
```

Comment:
Also, whatever the answer is, this will be the case with all components that are now using consul login to get an acl token.  They are all defining this preStop logout.

Reply:
Yeah we decided that we won't cleanup tokens in the case of a force-shutdown. The other option was to run a controller in k8s but it felt a bit heavy-handed for this and we thought we'll wait for user feedback before we implement it.